CONTRIBUTING.md

```diff
@@ -49,8 +49,8 @@ basically you’ll have to:
 * commit incrementally with readable and detailed commit messages
 * add the change to the `CHANGELOG.md` file if appropriated
 * submit a pull-request against the right branch of this repository:
-    * `dev` for new features
-    * `master` for bugfixes (unless only present in `dev`)
+    * `master` for new features or bug fixes for the current state
+    * `vX.Y` for bugfixes or backports on previous versions
```

Comment:
Shall we refer to `vMAJOR.MINOR` to reflect the use of semver?

Reply:
We don't follow semver (and we never had). I fixed the reference to semver to the rightful PEP 440

Reply:
Ah, this is stated in the `versionning.md` documentation file. Maybe worth referencing the right link and versionning scheme:

> UData uses [semantic versionning][semver] to publish its published releases.
> ...
> [semver]: http://semver.org/